python/README.md

```diff
@@ -16,72 +32,68 @@ poetry install
 ```
 
-Make sure you have an
-[Open AI API Key](https://openai.com/api/) or
-[Azure Open AI service key](https://learn.microsoft.com/azure/cognitive-services/openai/quickstart?pivots=rest-api)
 
-Copy those keys into a `.env` file in this repo (see the `.env.example` file):
+# Get Started with Semantic Kernel ⚡
 
-```
-OPENAI_API_KEY=""
-OPENAI_ORG_ID=""
-AZURE_OPENAI_API_KEY=""
-AZURE_OPENAI_ENDPOINT=""
-```
-
-### Quickstart ⚡
+## Example: Running a simple prompt.
 
 ```python
 import semantic_kernel as sk
-import semantic_kernel.ai.open_ai as sk_oai
+from semantic_kernel.ai.open_ai import OpenAITextCompletion
 
 kernel = sk.create_kernel()
 
-# This requires a `.env` file in your current
-# directory (see above)
+# Prepare OpenAI backend using credentials stored in the `.env` file
 api_key, org_id = sk.openai_settings_from_dot_env()
+kernel.config.add_text_backend("dv", OpenAITextCompletion("text-davinci-003", api_key, org_id))
 
-kernel.config.add_text_backend(
-    "davinci-002", sk_oai.OpenAITextCompletion(
-        "text-davinci-002", api_key, org_id
-    )
-)
-
-sk_prompt = """
-{{$input}}
+# Wrap your prompt in a function
+prompt = kernel.create_semantic_function("""
+1) A robot may not injure a human being or, through inaction,
+allow a human being to come to harm.
 
-Give me the TLDR in exactly 5 words.
-"""
+2) A robot must obey orders given it by human beings except where
+such orders would conflict with the First Law.
 
-text_to_summarize = """
-    1) A robot may not injure a human being or, through inaction,
-    allow a human being to come to harm.
+3) A robot must protect its own existence as long as such protection
+does not conflict with the First or Second Law.
 
-    2) A robot must obey orders given it by human beings except where
-    such orders would conflict with the First Law.
+Give me the TLDR in exactly 5 words.""")
 
-    3) A robot must protect its own existence as long as such protection
-    does not conflict with the First or Second Law.
-"""
-
-tldr_function = kernel.create_semantic_function(
-    sk_prompt,
-    max_tokens=200,
-    temperature=0,
-    top_p=0.5,
-)
+# Run your prompt
+print(prompt()) # => Robots must not harm humans.
+```
 
-summary = await kernel.run_on_str_async(text_to_summarize, tldr_function)
-output = str(summary.variables).strip()
-print("Output: " + output)
+## Example: Turn prompts into **reusable functions** with input parameters.
 
-# Output: Robots must not harm humans.
+```python
+# Create a reusable function with one input parameter
+summarize = kernel.create_semantic_function("{{$input}}\n\nOne line TLDR with the fewest words.")
+
+# Summarize the laws of thermodynamics
+print(summarize("""
+1st Law of Thermodynamics - Energy cannot be created or destroyed.
+2nd Law of Thermodynamics - For a spontaneous process, the entropy of the universe increases.
+3rd Law of Thermodynamics - A perfect crystal at zero Kelvin has zero entropy."""))
+
+# Summarize the laws of motion
```

Comment:
I think just having 1 example, not 3, is enough to illustrate the point. You'll have more space to highlight other examples.

Reply:
answered in other comment